calblueprint/berkeley-student-cooperative
# Andrei/user auth

# ✨ New in this PR ✨

## One liner: what did you do? 🛠
User Auth:  Sign in, Register, Sign out


## Coverage 🙆‍♀️
_use this section to break up your task into submodules and track progress_ 
 - [x] Sign in with email + password
 - [x] Register with email, name, password (missing CSV readability)
 - [x] Signout
 - [x] Uses addUser and getUser queries to create and retrieve user documents from FB
 - [x] UserContext helper function (meant to be plugged into that sprint)

## How can the reviewer test your code? 👩‍💻
Basic buttons: SIGN IN, Register, signout, Current signed-in user.  Change the values of email, password, name for register or sign in buttons in the index.tsx file.  Check that 1. A new user is created on the auth page in FB and 2. a user document is created in the database.


## Any bugs you encountered or still having trouble with? 🐛


## Resources 📔
_link online resources and related PRs_ 
[setUpUsers1](https://github.com/calblueprint/berkeley-student-cooperative/pull/3)
and userContext.



🏚 cc: @AndreiLTan
